tests/Functional/ContentProjectRenderTest.php

```diff
@@ -263,6 +263,15 @@ public function testProjectRendering(): void {
     $this->assertEquals($stakeholder_sub_headers[0]->getText(), 'Coordinators');
     $this->assertStakeholderOrganisationRendering($project_stakeholders, 'coordinator');
 
+    // Create image for a new logo that is unpublished and assert that is not
+    // rendered.
+    $media = $this->createMediaImage('coordinator_logo');
```

Comment:
Here as well, you are creating a new one. An existing one should be asserted that it shows, then unpublish, then assert it no longer shows.